src/juxt/site/alpha/util.clj

```diff
@@ -20,6 +20,15 @@
            (assoc-some m k v)
            (partition 2 kvs))))
 
+(defn assoc-some-2
```

Comment:
I think the above assoc-some is only used in one place so I would just make the fix there rather than adding a second one. I have no idea why it wasn't associng false values, we use booleans on several projects and haven't had issues before, I guess the key being missing is always treated the same as it being present and false so no one noticed/cared.

But I think I agree this is better

Reply:
Forgot to add, I originally made the change in `assoc-some` but that broke all queries with an exception (that I forgot to grab)

Reply:
Updated the PR to not have to duplicate `assoc-some` results in a cleaner and smaller change